docs/src/reference.md

```diff
@@ -1,4 +1,17 @@
 # Reference
-
+​
+## Contents
+​
+```@contents
+Pages = ["reference.md"]
+```
+​
+## Index
+​
 ```@index
+Pages = ["reference.md"]
 ```
+​
+```@autodocs
+Modules = [RipQP]
```

Comment:
I'm not sure that @autodocs will keep the structure. Is it possible to change to this (I can do it in another PR if needed):

```

> ## RipQP
> 
> ```@docs
> ripqp
> ```
> 
> ## Input Types
> 
> ```@docs
> InputConfig
> InputTol
> ```
> 
> ## Solvers
> 
> ```@docs
> SolverParams
> K2LDLParams
> K2_5LDLParams
> ```

```